src/network/clientpackethandler.cpp

```diff
@@ -565,10 +565,15 @@ void Client::handleCommand_HP(NetworkPacket *pkt)
 		m_script->on_hp_modification(hp);
 
 	if (hp < oldhp) {
-		// Add to ClientEvent queue
 		ClientEvent *event = new ClientEvent();
 		event->type = CE_PLAYER_DAMAGE;
 		event->player_damage.amount = oldhp - hp;
+		event->player_damage.effect = true;
+		// Wrap effect extraction within a
+		// try-catch to preserve backwards compat
```

Comment:
should be obvious to the reader, but even if not it doesn't make sense to put this comment everywhere the try-catch idom is used
```suggestion
```